apps/genrsa.c

```diff
@@ -121,8 +121,12 @@ int genrsa_main(int argc, char **argv)
     argv = opt_rest();
 
     if (argc == 1) {
-        if (!opt_int(argv[0], &num) || num <= 0)
+        if (!opt_int(argv[0], &num) || num <= 0) {
             goto end;
+        } else if (num > 16384) {
```

Comment:
This does not need to be an `else if`, a simple `else` can be used instead, avoiding the need to add braces to the previous `if`